doc/_static/css/custom.css

```diff
@@ -7,15 +7,64 @@ body {
     .toctree-wrapper > ul {
         columns: 2;
     }
+    .toc-outside-links {
+        columns: 2;
+    }
+}
+.item-framing-box {
```

Comment:
1. Can we add a brief comment above this group of rules to explain what they're doing and why? It could also link the issue / PR to avoid having the extra step of git blame.
2. Is there a more specific prefix we can use than `item-framing-`?

Reply:
I renamed some things, moved some things around to group them more nicely, and added comments. Hope it's more understandable now. Let me know if I should make some more changes. There isn't a github star button change in this PR, I just edited the size and position for visual purposes.